src/AppBundle/Model/Traits/TitleUserTrait.php

```diff
@@ -51,6 +51,10 @@ public function getDisplayTitle()
      */
     public function setTitle($title)
     {
+        // Remove 4-byte unicode characters (replace with the "replacement character" �).
+        // Kudos https://stackoverflow.com/a/24672780
+        // More info http://unicode.org/reports/tr36/#Deletion_of_Noncharacters
+        $title = preg_replace('/[\x{10000}-\x{10FFFF}]/u', "\xEF\xBF\xBD", $title);
```

Comment:
If we know we might need to do this with more strings in the future, would we be better off with a function available broadly that returns the corrected string?